README.md

```diff
@@ -56,6 +56,7 @@ __AWESOME__ apps counter: __253__ 🎉
 - [Download Manager](#download-manager)
 - [Drawing](#drawing)
 - [E-book Reader](#e-book-reader)
+- [Education](#education)
```

Comment:
We have a [Knowledge](https://github.com/Psyhackological/AAA?tab=readme-ov-file#knowledge) section already. I think it suits there.

Reply:
Hello) Thanks, fixed it)